cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/kubernetes/DeleteConfigMapOperation.java

```diff
@@ -43,8 +44,8 @@ public void execute(Vertx vertx, K8SUtils k8s, Handler<AsyncResult<Void>> handle
                         handler.handle(Future.succeededFuture());
                     }
                     else {
-                        log.error("Configmap deletion failed: {}", res.result());
-                        handler.handle(Future.failedFuture((Exception)res.result()));
+                        log.error("Configmap deletion failed: {}", res.cause().toString());
```

Comment:
@ppatierno it wasn't clear what logging was intended: Because there's a `{}` in the format string but the last parameter was a `Throwable` slf4j was actually printing a literal `{}` in the exception message, and dumping the stacktrace too. If dumping the stacktrace is wanted then you should omit a `{}` placeholder for the exception: slf4j will then dump the stacktrace. 

But actually this handler might be refactorable further, consider:

```java
                res -> {
                    if (res.succeeded()) {
                        log.info("{} {} has been created", resourceKind, resource);
                        handler.handle(Future.succeededFuture());
                    }
                    else {
                        log.error("{} creation failed: {}", resourceKind, res.cause().toString());
                        handler.handle(Future.failedFuture(res.cause()));
                    }
                }
```

We're effectively passing the `res` to the `handler` in both branches, right? All we're adding here is logging, but it's actually likely that the `handler` will end up logging the error too. So maybe all we're doing here is causing double logging, and we should consider just passing the `handler` directly as the 2nd argument to `executeBlocking()`, instead of having this lambda at all.